Model/src/main/java/org/gusdb/wdk/model/user/dataset/event/UserDatasetEventCleanupHandler.java

```diff
@@ -37,24 +33,12 @@ public UserDatasetEventCleanupHandler(
    */
   @Override
   public boolean shouldHandleEvent(EventRow row) {
```

Comment:
add a comment explaining why this is always true

Reply:
updated